cmd/manager/main.go

```diff
@@ -125,6 +127,11 @@ func newRootCommand() *cobra.Command {
 				log.Fatal(err)
 			}
 
+			// Start http server which will enable the /debug/health handler from go-healthcheck
+			log.Printf("Starting debug/health endpoint.")
```

Comment:
Any reason this is using printf vs one of the normal log levels?

Reply:
All the other “starting up” lines were printf. I can info log them.

Reply:
Updated!

Reply:
For reference, the difference between `Printf` and `Infof` is that `Printf` gets output to the log regardless of what the log level is set to. They both yield an info-level log entry, though.